packit_service/worker/handlers/testing_farm.py

```diff
@@ -393,6 +393,7 @@ def run(self) -> TaskResults:
         status_reporter = StatusReporter.get_instance(
             project=self.project,
             commit_sha=self.data.commit_sha,
+            packit_user=self.service_config.get_github_account_name(),
```

Comment:
Could you please check what is the account name? IIRC it might be prefixed with `app/`, but on GitLab, we »do not« have that prefix.

Reply:
I am not sure if I follow what you mean by the `app/` prefix. The [method](https://github.com/packit/packit-service/blob/50e1c9cd220ab4c3f875b1bda15dd701bd98180e/packit_service/config.py#L268) just returns one of `packit-as-a-service[bot]`, `packit-as-a-service-stg[bot]`, `packit-as-a-service-dev[bot]` depending on the deployment used. This should match the commenter name in the Github PRs/commits